### PR TITLE
Update vSphere CSI to v2.2.1

### PR DESCRIPTION
**Pull request by PennyScissors**
Update vSphere CSI chart to upstream's latest v2.2.1 (This is just image updates addressing CVEs, see references to issues below)

image-mirror PR: https://github.com/rancher/image-mirror/pull/137

### CODE REVIEW

**Review by jiaqiluo — APPROVED** [review]
LGTM

**Comment by PennyScissors** [comment]
@cbron Do I need to backport this update to 2.5?

**Review by aiyengar2 — APPROVED** [review]
LGTM

File: charts/rancher-vsphere-csi/rancher-vsphere-csi/100.0.0/values.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@ csiController:
     enabled: false
```
Comment (cbron) [inline charts/rancher-vsphere-csi/rancher-vsphere-csi/100.0.0/values.yaml:25]: Old asset should get deleted and index needs updating.

Comment (aiyengar2) [reply charts/rancher-vsphere-csi/rancher-vsphere-csi/100.0.0/values.yaml:25]: @cbron the version isn't changing so the index.yaml doesn't need to be modified. The asset also got modified, which means that it did change

**Review by cbron — APPROVED** [review]
lgtm, but versioning may change based on discussion.

**Pull request merged**